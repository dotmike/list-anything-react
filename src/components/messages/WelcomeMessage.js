import React from "react";
import { Message } from "semantic-ui-react";

const WelcomeMessage = () => (
  <Message info>
    <Message.Header>Welcome to ListAnything!</Message.Header>
  </Message>
);

export default WelcomeMessage;
