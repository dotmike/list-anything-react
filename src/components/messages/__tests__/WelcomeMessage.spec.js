import React from "react";
import { shallow } from "enzyme";

import WelcomeMessage from "../WelcomeMessage";

describe("WelcomeMessage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<WelcomeMessage />);
    expect(wrapper).toMatchSnapshot();
  });
});
