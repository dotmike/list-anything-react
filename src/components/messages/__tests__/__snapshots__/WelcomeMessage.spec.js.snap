// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeMessage renders correctly 1`] = `
<Message
  info={true}
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <MessageHeader>
    Welcome to ListAnything!
  </MessageHeader>
</Message>
`;
